### PR TITLE
switch to cheap-ruler [not ready]

### PR DESCRIPTION
**Pull request by tcql**
per #3 

cc @mourner

### CODE REVIEW

File: probematch.js [hunk 1]
```diff
@@ -26,13 +24,14 @@ module.exports = function (inputLines, opts) {
 
   for (var i = 0; i < lines.features.length; i++) {
     var coords = lines.features[i].geometry.coordinates;
+    var ruler = cheapRuler(coords[0][1], 'kilometers');
```
Comment (mourner) [inline probematch.js]: The idea is to create the ruler once for a tile, not on every coordinate.

Comment (tcql) [inline probematch.js]: :+1: makes sense

**Comment by tcql** [comment]
switched to cheap-ruler, did some code extraction, now this runs ~490 ops/sec vs prior 280 in a simple benchmark

**Comment by aaronlidman** [comment]
@tcql - Was there anything else needed here?

**Comment by tcql** [comment]
@aaronlidman nope, I think it was in good shape, just got lost

**Pull request merged**